webcompat/webhooks/helpers.py

```diff
@@ -116,7 +122,10 @@ def extract_priority_label(body):
                    for i, subpart in enumerate(subparts)
                    if 0 < i < hostname.count('.')]
         for domain in domains:
-            for site in site_db.query(Site).filter_by(url=domain):
+            for site in regional_site_db.query(SiteRegional).filter_by(url=domain):     # noqa
+                return f'priority-{priorities[site.priority - 1]}'
+
+            for site in global_site_db.query(SiteGlobal).filter_by(url=domain):
```

Comment:
The fact that we do it here again makes me think there is an opportunity for a function to reduce it and test it.